pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -709,6 +722,92 @@ func (h *Helper) updateConfig(configDir string, opt *StartOptions) error {
 		cfg.AdmissionConfig.PluginConfig[defaultsapi.BuildDefaultsPlugin] = buildDefaultsConfig
 	}
 
+	if opt.ServiceCatalog {
+		cfg.TemplateServiceBrokerConfig = &configapi.TemplateServiceBrokerConfig{
+			TemplateNamespaces: []string{"openshift"},
```

Comment:
there's a const openshiftNamespace but it might be located in the wrong package (pkg/bootstrap/docker) - maybe not this PR